knowledge_repo/app/static/css/pages/base.css

```diff
@@ -47,44 +68,73 @@
 }
 #navbar .navbar-form .input-group #searchbar {
     width: 100%;
-    background: #999;
-    color: #fff;
+    background: #ececec;
+    color: #000;
+    opacity: 1.0;
     border: none;
     transition: background 0.2s;
 }
 #navbar .navbar-form .input-group #searchbar:focus {
     background: #aaa;
     box-shadow: none;
     transition: background 0.3s;
+    background: #ececec;
 }
 #navbar .navbar-form .btn-default {
     border-color: whitesmoke;
+    background-color: #ececec;
+    border-color: #ececec;
 }
 #navbar .navbar-form .input-group #searchbar::-webkit-input-placeholder { /* Chrome */
-  color: #fff;
+  color: #000;
```

Comment:
use `black`. I'll stop commenting this from here on out, but please update the remaining cases